pkg/asset/manifests/azure/cluster.go

```diff
@@ -77,24 +82,151 @@ func GenerateClusterAssets(installConfig *installconfig.InstallConfig, clusterID
 						Type: capz.Internal,
 					},
 				},
+				ControlPlaneOutboundLB: &capz.LoadBalancerSpec{
+					FrontendIPsCount: to.Ptr(int32(1)),
```

Comment:
```suggestion
					FrontendIPsCount: ptr.to[int32](1),
```

Reply:
I am okay changing this, but do you mind explaining to me why one way is preferential to the other?

Reply:
`to.Ptr` is from the azure lib ("github.com/Azure/azure-sdk-for-go/sdk/azcore/to") but you're already importing `"k8s.io/utils/ptr"` so I thought it'd be better to stay consistent.